### ISSUE TITLE
bug: home-manager install error on Arch Linux

### ISSUE THREAD

**Issue by D3vil0p3r** [nix-community/home-manager#4405]
### Are you following the right branch?

- [X] My Nixpkgs and Home Manager versions are in sync

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue description

On Arch Linux I installed nix by `sudo pacman -S nix` (version 2.17.0-3) and I followed [this](https://nix-community.github.io/home-manager/index.html#sec-install-standalone) docs to install **home-manager**. 

I run:
```
sudo pacman -S nix
sudo systemctl enable nix-daemon.service
sudo systemctl start nix-daemon.service

nix-channel --add https://nixos.org/channels/nixpkgs-unstable nixpkgs
nix-channel --add https://github.com/nix-community/home-manager/archive/master.tar.gz home-manager
nix-channel --update

nix-shell '<home-manager>' -A install
```
and I get:
```
this derivation will be built:
  /nix/store/s2vfjdb7lnb69yvqy2nn5vnrc6w20mr2-home-manager.drv
these 78 paths will be fetched (28.60 MiB download, 129.05 MiB unpacked):
  /nix/store/l9ygjsiicb4p9isxgjqand0049rsqvhg-acl-2.3.1
  /nix/store/ix8bn3420zkyrkirb1aq0ymisil0m216-attr-2.5.1
  /nix/store/9295khdqlnb3c60g3xwbanwax6wdxihf-aws-c-auth-0.7.0
  /nix/store/9s6mpa042g5dn2bgkazpbsqaw1rdpc2p-aws-c-cal-0.6.0
  /nix/store/7izc22k2bz3qmz330ac7d6sp6ny6p704-aws-c-common-0.8.23
  /nix/store/5fir2bdjnjw34avj2fyf4h6dak5jlpc4-aws-c-compression-0.2.17
  /nix/store/dr1wdfp4r1l8hm0w7m1j8rvnn7gvkziw-aws-c-event-stream-0.3.1
  /nix/store/ssl01hj9dqk302829my1fc1dmi94b9g1-aws-c-http-0.7.11
  /nix/store/szj17ryjsxsgxvqgjwwdmrrivml8s3q7-aws-c-io-0.13.29
  /nix/store/x4x1q3wdxr3wqws611fpjnsb0i07g086-aws-c-mqtt-0.8.14
  /nix/store/n5pyl0rcfcjyd0h2s3lr2yl737krmqvf-aws-c-s3-0.3.13
  /nix/store/lypcnwmpmxrzz8qzjh8ji4lh4a16yhh4-aws-c-sdkutils-0.1.11
  /nix/store/mj6aagvl2hmlaa5livklhv6ncnhbf8xf-aws-checksums-0.1.17
  /nix/store/r6smw3k3sif8j5p7mgbsgznja1dwmhic-aws-crt-cpp-0.20.3
  /nix/store/kqvvzvzmyx37g3ymxlp7c608r222mg9b-aws-sdk-cpp-1.11.118
  /nix/store/r4vxljid3iq94jp7qvd639sps0fscwy3-bash-5.2-p15
  /nix/store/anvg6mpa8q2lvhp1h0yr3yn960zskrfj-bash-interactive-5.2-p15
  /nix/store/3i4nn98sqhlnzj7lrma3j2y13qpk1ib0-boehm-gc-8.2.2
  /nix/store/1f2aahqnd71c4fwggizkg75vav38zqc3-brotli-1.0.9-lib
  /nix/store/ip34qcxkyslm8z2jl4q7w8idykmcxgm2-busybox-static-x86_64-unknown-linux-musl-1.36.1
  /nix/store/pjk8v6xrvfdv7mb7ahsrip5scgf6jbif-bzip2-1.0.8
  /nix/store/ir3achw4jh3qyyac0vkj8jb46l06psqh-bzip2-1.0.8-bin
  /nix/store/5wspmqhgsfs7dza9gcyrj5yxbyd00q7y-coreutils-9.3
  /nix/store/fd98bpjcmx4nb872sdb7mmar4la5m5sj-curl-8.2.1
  /nix/store/pmkqwxzi27lzdzkh2zpqajks9b3521jz-diffutils-3.10
  /nix/store/grcz57p25iazrizgny4saixb17bn73z3-ed-1.19
  /nix/store/k7m4s3j3ikd48zz017xr0zn14wif4k9x-editline-1.17.1
  /nix/store/yyrljddc4wqraxs72d8kzcf3filcncky-file-5.45
  /nix/store/039nii0j16pkzabxi9nm5dkh0z8awaww-findutils-4.9.0
  /nix/store/scv8hzzf0vpxaqj96qjhzdcf04aygga1-gawk-5.2.2
  /nix/store/ci51zm09w9skb92zkc5x9x2vr1pkb0h6-gcc-12.3.0-lib
  /nix/store/yyfqqmdx3s7w088dqy0qhyqd0gjcvm2k-gcc-12.3.0-libgcc
  /nix/store/lhrsbc2g0q0jhqvws33031bvxfpjb57v-gettext-0.21.1
  /nix/store/9la894yvmmksqlapd4v16wvxpaw3rg70-glibc-2.37-8
  /nix/store/dyqxh02ccis198w7crrwn519z3fyxcwv-gmp-with-cxx-6.3.0
  /nix/store/nzxlaxndhjcxk9xxas0gwiywb74zw7a7-gnu-config-2023-07-31
  /nix/store/m88w8in2zf5dwijxypyba02f0ff260rm-gnugrep-3.11
  /nix/store/k51cq3ap20c69wffjsl8qjs21wy5wlrq-gnumake-4.4.1
  /nix/store/5ccllrgswk61a44yl7jcxs20jql1y32b-gnused-4.9
  /nix/store/ygjmhqdm8labz14xxscz003srl1p2c94-gnutar-1.35
  /nix/store/pm6pfypn61d9n9gy19i01ak8hwc6g688-gzip-1.12
  /nix/store/a4lx1h4zzb3a2r6qkj8bxk4xcik17xrs-hostname-net-tools-2.10
  /nix/store/9xx67wl66zx277vm55ksnazn1xhkgrb3-jq-1.6-bin
  /nix/store/00dhmy732slsm1lm68vw6d161l1b8p7h-jq-1.6-lib
  /nix/store/6xa6wyzbby9wzgzqxqiw7ld46pr1g6bc-keyutils-1.6.3-lib
  /nix/store/cf6ibrgifvah83xn5vfyx3s2bddzq5sc-less-633
  /nix/store/mkcy84mqj59pr3mpjr0gz39qiqravc4w-libarchive-3.6.2-lib
  /nix/store/q7ncxx2ikx1mcwgv61099bgdmx795ij8-libcpuid-0.6.3
  /nix/store/7lihz4s5pg8irvp691x94argri346mls-libidn2-2.3.4
  /nix/store/knpyd8vysz5zvpbm4ak02bhwxrngw6zv-libkrb5-1.20.1
  /nix/store/i3xnk55cjs41xlw0vqd3l2ig1mldxah9-libseccomp-2.5.4-lib
  /nix/store/3l4ff72kgpfiq01k38cj5ia6q3ngl4lp-libsodium-1.0.18
  /nix/store/r0zzy4hx48mrwrq6vac82nyyfhngppr7-libssh2-1.11.0
  /nix/store/p58fbj416fzm72h2h2v87h7a8brl55d5-libunistring-1.1
  /nix/store/57ilg207pwyynwbnfggwdvgy6yjpg25a-libxml2-2.11.4
  /nix/store/q9scsfap3s8xdh9pqdsc1lfmp80zsbhy-lowdown-1.0.2-lib
  /nix/store/n31yg2ngxgb1smg553q4kak5ckf9cnb3-ncurses-6.4
  /nix/store/58v9ayh73z4wmziy6ciksw7g6pckd4g7-net-tools-2.10
  /nix/store/88hwnm9gys7rp1yaghyhmlxn4yjr8grx-nghttp2-1.54.0-lib
  /nix/store/w5dfnh4j5d3d8z2c5i8999dyqw1wcaqi-nix-2.15.2
  /nix/store/lvrd4xj0ydcssfw5k1s17r7fmayshiy7-nix-2.15.2-man
  /nix/store/kdznljn883lb5xw2390aqyp9zy4zvl72-nixos-option
  /nix/store/d6h0nf1fbx3b4bxyx61brwdd5jmgl1mk-nlohmann_json-3.11.2
  /nix/store/32ab6x8qfxck1hx4lg1h0i38jr1rw5hp-oniguruma-6.9.8-lib
  /nix/store/dfmd2p745k6h3rfzzy3cxnvk2bza63vy-openssl-3.0.10
  /nix/store/l6wb1grmnawfvy6xqp5x29jsq9p5dym8-patch-2.7.6
  /nix/store/dhg99sq2invv3scrlv6abxsb5pcz8ip6-patchelf-0.15.0
  /nix/store/k1129sixmk24qqc30paf9adx895y9mnz-pcre2-10.42
  /nix/store/8sxqidx9fks956gizdg9w9ndmffddvwh-readline-8.2p1
  /nix/store/flnrym1vjyfz9a5jhyszxmih8i223zim-s2n-tls-1.3.48
  /nix/store/s71iqh14mpkga588y5fvsv13d7qdpz45-sqlite-3.42.0
  /nix/store/80fgzsxy3x8gyv9kmjdhmis26grm2adi-stdenv-linux
  /nix/store/8sc724vl78ig662fzlxzg4i6g6m4pf4q-update-autotools-gnu-config-scripts-hook
  /nix/store/ibw4h4lp57820gi5i46iwdnp9i58k8li-xgcc-12.3.0-libgcc
  /nix/store/x25wbsvsbnd3xj52pgysblnqgriqg7hr-xz-5.4.4
  /nix/store/ikzmh72n5f785gkfm02kzg4cl8d7hzmg-xz-5.4.4-bin
  /nix/store/i039srr9s4li309rclbm3421ifavgl0g-zlib-1.2.13
  /nix/store/dvs7qzvzhp1ff2n17ypih5pjhqjv93j8-zstd-1.5.5
copying path '/nix/store/nzxlaxndhjcxk9xxas0gwiywb74zw7a7-gnu-config-2023-07-31' from 'https://cache.nixos.org'...
copying path '/nix/store/ibw4h4lp57820gi5i46iwdnp9i58k8li-xgcc-12.3.0-libgcc' from 'https://cache.nixos.org'...
copying path '/nix/store/yyfqqmdx3s7w088dqy0qhyqd0gjcvm2k-gcc-12.3.0-libgcc' from 'https://cache.nixos.org'...
copying path '/nix/store/p58fbj416fzm72h2h2v87h7a8brl55d5-libunistring-1.1' from 'https://cache.nixos.org'...
copying path '/nix/store/ip34qcxkyslm8z2jl4q7w8idykmcxgm2-busybox-static-x86_64-unknown-linux-musl-1.36.1' from 'https://cache.nixos.org'...
copying path '/nix/store/lvrd4xj0ydcssfw5k1s17r7fmayshiy7-nix-2.15.2-man' from 'https://cache.nixos.org'...
copying path '/nix/store/d6h0nf1fbx3b4bxyx61brwdd5jmgl1mk-nlohmann_json-3.11.2' from 'https://cache.nixos.org'...
copying path '/nix/store/8sc724vl78ig662fzlxzg4i6g6m4pf4q-update-autotools-gnu-config-scripts-hook' from 'https://cache.nixos.org'...
copying path '/nix/store/7lihz4s5pg8irvp691x94argri346mls-libidn2-2.3.4' from 'https://cache.nixos.org'...
copying path '/nix/store/9la894yvmmksqlapd4v16wvxpaw3rg70-glibc-2.37-8' from 'https://cache.nixos.org'...
copying path '/nix/store/ix8bn3420zkyrkirb1aq0ymisil0m216-attr-2.5.1' from 'https://cache.nixos.org'...
copying path '/nix/store/7izc22k2bz3qmz330ac7d6sp6ny6p704-aws-c-common-0.8.23' from 'https://cache.nixos.org'...
copying path '/nix/store/1f2aahqnd71c4fwggizkg75vav38zqc3-brotli-1.0.9-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/grcz57p25iazrizgny4saixb17bn73z3-ed-1.19' from 'https://cache.nixos.org'...
copying path '/nix/store/r4vxljid3iq94jp7qvd639sps0fscwy3-bash-5.2-p15' from 'https://cache.nixos.org'...
copying path '/nix/store/k7m4s3j3ikd48zz017xr0zn14wif4k9x-editline-1.17.1' from 'https://cache.nixos.org'...
copying path '/nix/store/scv8hzzf0vpxaqj96qjhzdcf04aygga1-gawk-5.2.2' from 'https://cache.nixos.org'...
copying path '/nix/store/ci51zm09w9skb92zkc5x9x2vr1pkb0h6-gcc-12.3.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/k51cq3ap20c69wffjsl8qjs21wy5wlrq-gnumake-4.4.1' from 'https://cache.nixos.org'...
copying path '/nix/store/pjk8v6xrvfdv7mb7ahsrip5scgf6jbif-bzip2-1.0.8' from 'https://cache.nixos.org'...
copying path '/nix/store/5ccllrgswk61a44yl7jcxs20jql1y32b-gnused-4.9' from 'https://cache.nixos.org'...
copying path '/nix/store/6xa6wyzbby9wzgzqxqiw7ld46pr1g6bc-keyutils-1.6.3-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/q7ncxx2ikx1mcwgv61099bgdmx795ij8-libcpuid-0.6.3' from 'https://cache.nixos.org'...
copying path '/nix/store/i3xnk55cjs41xlw0vqd3l2ig1mldxah9-libseccomp-2.5.4-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/q9scsfap3s8xdh9pqdsc1lfmp80zsbhy-lowdown-1.0.2-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/3l4ff72kgpfiq01k38cj5ia6q3ngl4lp-libsodium-1.0.18' from 'https://cache.nixos.org'...
copying path '/nix/store/n31yg2ngxgb1smg553q4kak5ckf9cnb3-ncurses-6.4' from 'https://cache.nixos.org'...
copying path '/nix/store/l9ygjsiicb4p9isxgjqand0049rsqvhg-acl-2.3.1' from 'https://cache.nixos.org'...
copying path '/nix/store/58v9ayh73z4wmziy6ciksw7g6pckd4g7-net-tools-2.10' from 'https://cache.nixos.org'...
copying path '/nix/store/88hwnm9gys7rp1yaghyhmlxn4yjr8grx-nghttp2-1.54.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/32ab6x8qfxck1hx4lg1h0i38jr1rw5hp-oniguruma-6.9.8-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/ir3achw4jh3qyyac0vkj8jb46l06psqh-bzip2-1.0.8-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/dfmd2p745k6h3rfzzy3cxnvk2bza63vy-openssl-3.0.10' from 'https://cache.nixos.org'...
copying path '/nix/store/l6wb1grmnawfvy6xqp5x29jsq9p5dym8-patch-2.7.6' from 'https://cache.nixos.org'...
copying path '/nix/store/k1129sixmk24qqc30paf9adx895y9mnz-pcre2-10.42' from 'https://cache.nixos.org'...
copying path '/nix/store/x25wbsvsbnd3xj52pgysblnqgriqg7hr-xz-5.4.4' from 'https://cache.nixos.org'...
copying path '/nix/store/i039srr9s4li309rclbm3421ifavgl0g-zlib-1.2.13' from 'https://cache.nixos.org'...
copying path '/nix/store/5fir2bdjnjw34avj2fyf4h6dak5jlpc4-aws-c-compression-0.2.17' from 'https://cache.nixos.org'...
copying path '/nix/store/lypcnwmpmxrzz8qzjh8ji4lh4a16yhh4-aws-c-sdkutils-0.1.11' from 'https://cache.nixos.org'...
copying path '/nix/store/mj6aagvl2hmlaa5livklhv6ncnhbf8xf-aws-checksums-0.1.17' from 'https://cache.nixos.org'...
copying path '/nix/store/knpyd8vysz5zvpbm4ak02bhwxrngw6zv-libkrb5-1.20.1' from 'https://cache.nixos.org'...
copying path '/nix/store/pm6pfypn61d9n9gy19i01ak8hwc6g688-gzip-1.12' from 'https://cache.nixos.org'...
copying path '/nix/store/ygjmhqdm8labz14xxscz003srl1p2c94-gnutar-1.35' from 'https://cache.nixos.org'...
copying path '/nix/store/00dhmy732slsm1lm68vw6d161l1b8p7h-jq-1.6-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/a4lx1h4zzb3a2r6qkj8bxk4xcik17xrs-hostname-net-tools-2.10' from 'https://cache.nixos.org'...
copying path '/nix/store/yyrljddc4wqraxs72d8kzcf3filcncky-file-5.45' from 'https://cache.nixos.org'...
copying path '/nix/store/s71iqh14mpkga588y5fvsv13d7qdpz45-sqlite-3.42.0' from 'https://cache.nixos.org'...
copying path '/nix/store/57ilg207pwyynwbnfggwdvgy6yjpg25a-libxml2-2.11.4' from 'https://cache.nixos.org'...
copying path '/nix/store/ikzmh72n5f785gkfm02kzg4cl8d7hzmg-xz-5.4.4-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/9xx67wl66zx277vm55ksnazn1xhkgrb3-jq-1.6-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/m88w8in2zf5dwijxypyba02f0ff260rm-gnugrep-3.11' from 'https://cache.nixos.org'...
copying path '/nix/store/dhg99sq2invv3scrlv6abxsb5pcz8ip6-patchelf-0.15.0' from 'https://cache.nixos.org'...
copying path '/nix/store/dvs7qzvzhp1ff2n17ypih5pjhqjv93j8-zstd-1.5.5' from 'https://cache.nixos.org'...
copying path '/nix/store/3i4nn98sqhlnzj7lrma3j2y13qpk1ib0-boehm-gc-8.2.2' from 'https://cache.nixos.org'...
copying path '/nix/store/lhrsbc2g0q0jhqvws33031bvxfpjb57v-gettext-0.21.1' from 'https://cache.nixos.org'...
copying path '/nix/store/dyqxh02ccis198w7crrwn519z3fyxcwv-gmp-with-cxx-6.3.0' from 'https://cache.nixos.org'...
copying path '/nix/store/r0zzy4hx48mrwrq6vac82nyyfhngppr7-libssh2-1.11.0' from 'https://cache.nixos.org'...
copying path '/nix/store/9s6mpa042g5dn2bgkazpbsqaw1rdpc2p-aws-c-cal-0.6.0' from 'https://cache.nixos.org'...
copying path '/nix/store/flnrym1vjyfz9a5jhyszxmih8i223zim-s2n-tls-1.3.48' from 'https://cache.nixos.org'...
copying path '/nix/store/cf6ibrgifvah83xn5vfyx3s2bddzq5sc-less-633' from 'https://cache.nixos.org'...
copying path '/nix/store/8sxqidx9fks956gizdg9w9ndmffddvwh-readline-8.2p1' from 'https://cache.nixos.org'...
copying path '/nix/store/mkcy84mqj59pr3mpjr0gz39qiqravc4w-libarchive-3.6.2-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/fd98bpjcmx4nb872sdb7mmar4la5m5sj-curl-8.2.1' from 'https://cache.nixos.org'...
copying path '/nix/store/anvg6mpa8q2lvhp1h0yr3yn960zskrfj-bash-interactive-5.2-p15' from 'https://cache.nixos.org'...
copying path '/nix/store/5wspmqhgsfs7dza9gcyrj5yxbyd00q7y-coreutils-9.3' from 'https://cache.nixos.org'...
copying path '/nix/store/szj17ryjsxsgxvqgjwwdmrrivml8s3q7-aws-c-io-0.13.29' from 'https://cache.nixos.org'...
copying path '/nix/store/dr1wdfp4r1l8hm0w7m1j8rvnn7gvkziw-aws-c-event-stream-0.3.1' from 'https://cache.nixos.org'...
copying path '/nix/store/ssl01hj9dqk302829my1fc1dmi94b9g1-aws-c-http-0.7.11' from 'https://cache.nixos.org'...
copying path '/nix/store/pmkqwxzi27lzdzkh2zpqajks9b3521jz-diffutils-3.10' from 'https://cache.nixos.org'...
copying path '/nix/store/039nii0j16pkzabxi9nm5dkh0z8awaww-findutils-4.9.0' from 'https://cache.nixos.org'...
copying path '/nix/store/x4x1q3wdxr3wqws611fpjnsb0i07g086-aws-c-mqtt-0.8.14' from 'https://cache.nixos.org'...
copying path '/nix/store/9295khdqlnb3c60g3xwbanwax6wdxihf-aws-c-auth-0.7.0' from 'https://cache.nixos.org'...
copying path '/nix/store/n5pyl0rcfcjyd0h2s3lr2yl737krmqvf-aws-c-s3-0.3.13' from 'https://cache.nixos.org'...
copying path '/nix/store/r6smw3k3sif8j5p7mgbsgznja1dwmhic-aws-crt-cpp-0.20.3' from 'https://cache.nixos.org'...
copying path '/nix/store/80fgzsxy3x8gyv9kmjdhmis26grm2adi-stdenv-linux' from 'https://cache.nixos.org'...
copying path '/nix/store/kqvvzvzmyx37g3ymxlp7c608r222mg9b-aws-sdk-cpp-1.11.118' from 'https://cache.nixos.org'...
copying path '/nix/store/w5dfnh4j5d3d8z2c5i8999dyqw1wcaqi-nix-2.15.2' from 'https://cache.nixos.org'...
copying path '/nix/store/kdznljn883lb5xw2390aqyp9zy4zvl72-nixos-option' from 'https://cache.nixos.org'...
building '/nix/store/s2vfjdb7lnb69yvqy2nn5vnrc6w20mr2-home-manager.drv'...
install: creating directory '/nix/store/13g503spdk6m80zjs29hjdng5160cpw5-home-manager'
install: creating directory '/nix/store/13g503spdk6m80zjs29hjdng5160cpw5-home-manager/bin'
'/nix/store/lwz6xx15cbfygnv7s1vgvg8870pyr636-home-manager' -> '/nix/store/13g503spdk6m80zjs29hjdng5160cpw5-home-manager/bin/home-manager'
    .          ..             ┌───────── Hardware Information ─────────┐ 
   :####:     ####.            ​ 󰟀    VMware Virtual Platform None 
  .################            ​ 󰻠    Intel i7-8750H (4) @ 2.2GHz 
 :##################           ​ 󰢮    00:0f.0 VMware SVGA II Adapter 
.###################.          ​  (/)   24G / 100G (24%) 
########     #######           ​ 󰍛    1224MiB / 3881MiB (31%) 
#######  #### #####            ​ 󰹑    1714x874 
:#######.      ####           ├───────── Software Information ─────────┤ 
 #########  #   ##   #         ​     Athena OS 
 #######   ##      ####        ​     6.4.12-zen1-1-zen 
########  ####    #######      ​     GNOME 44.4 
########  #####   ########     ​     Mutter 
########  #######  #######     ​     fish 3.6.1 
 #######  ########  #######    ​     kitty 
 ########  #########  ######   ​ 󰅐    4 hours 
  ########  #########  #####  └──────────────── Colors ─────────────────┘ 
    #######  #########  ####                             
     #######  #########  ##
       #######  ######## ##
          ###### ######## #
               ### #######
                     ######
                        ####
                          ##

Creating /home/athena/.config/home-manager/home.nix...

Creating initial Home Manager generation...

/usr/include/c++/13.2.1/string_view:258: constexpr const std::basic_string_view<_CharT, _Traits>::value_type& std::basic_string_view<_CharT, _Traits>::operator[](size_type) const [with _CharT = char; _Traits = std::char_traits<char>; const_reference = const char&; size_type = long unsigned int]: Assertion '__pos < this->_M_len' failed.
/nix/store/13g503spdk6m80zjs29hjdng5160cpw5-home-manager/bin/home-manager: line 509: 35550 Aborted                 (core dumped) nix-build "<home-manager/home-manager/home-manager.nix>" "${extraArgs[@]}" "${PASSTHROUGH_OPTS[@]}" --argstr confPath "$HOME_MANAGER_CONFIG" --argstr confAttr "$HOME_MANAGER_CONFIG_ATTRIBUTE"
Uh oh, the installation failed! Please create an issue at

    https://github.com/nix-community/home-manager/issues

if the error seems to be the fault of Home Manager.
```

### System information

```markdown
- system: `"x86_64-linux"`
 - host os: `Linux 6.4.12-zen1-1-zen, Athena OS, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.0`
 - nixpkgs: `/home/athena/.nix-defexpr/channels/nixpkgs`
```

**Comment by Layerex** [comment]
Can confirm

**Comment by eclairevoyant** [comment]
relevant issue: NixOS/nix#8737
will be fixed in NixOS/nix#8936, so I guess just wait for the fix to be released on Arch
